Documentation/contributing/testing/e2e.rst

```diff
@@ -227,7 +229,7 @@ framework in the ``test/`` directory and interact with ginkgo directly:
       -cilium.timeout duration
             Specifies timeout for test run (default 24h0m0s)
 
-    Ginkgo ran 1 suite in 4.312100241s
+    Ginkgo ran 1 suite in 3.086476005s
```

Comment:
Unnecessary change?

Reply:
Side effect of copy&pasting the output :smile: I'll drop it if another revision is needed.

Reply:
Dropped the flag again as it's not necessary, we can just check the `RACE` env var at runtime.